src/v/raft/rpc_client_protocol.cc

```diff
@@ -75,10 +98,16 @@ rpc_client_protocol::install_snapshot(
           return client.install_snapshot(std::move(r), std::move(opts))
             .then(&rpc::get_ctx_data<install_snapshot_reply>);
       });
+    // NOLINTEND(bugprone-use-after-move)
```

Comment:
this one actually looks "unsafe". it's safe because timeout is a primitive type, the IIUC the c++14 semantic change doesn't protect this case because parameter evaluation order is still undefined. maybe here NOLINT should be removed and then before `return ...` have `timeout = opts.timeout` and use `timeout` in the parameter instead of access the opt field.